src/types.h

```diff
@@ -377,13 +377,26 @@ typedef struct
     char strings[];
 } process_message_t, *pprocess_message_t;
 
+#define MAX_PATH_SEG 64
```

Comment:
just so I understand this, this is just saying that the paths we allow to filter on must be less than 64 characters right? There could be a segment in the path that is bigger than that but that segment just won't be specified in the filter.

Reply:
Yes; we should still be able accept longer path segments via the wildcard matchers, but we can't distinguish path segments that don't differ until after their 63rd character in our globs.